packages/console/src/mdx-components/Tabs/index.tsx

```diff
@@ -0,0 +1,108 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Nullable } from '@silverhand/essentials';
+import React, { useState, isValidElement, type ReactElement, cloneElement } from 'react';
+
+import type { Props as TabItemProps } from '../TabItem';
+
+type Props = {
+  className?: string;
+  children: ReactElement<TabItemProps>;
+};
+
+// A very rough duck type, but good enough to guard against mistakes while
+// allowing customization
+function isTabItem(comp: ReactElement): comp is ReactElement<TabItemProps> {
```

Comment:
component

Reply:
Same as above. This is copied from docusaurus, and will improve this in [LOG-2331](https://linear.app/silverhand/issue/LOG-2331/improve-tabs-and-tabitem-components-in-mdx-components).

Reply:
cool